presto-product-tests/conf/hadoop/singlenode-viewfs/hdfs-site.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<configuration>
+
+    <property>
+        <name>dfs.namenode.name.dir</name>
+        <value>/var/lib/hadoop-hdfs/cache/name/</value>
+    </property>
+
+    <!-- Hdfs viewfs -->
+    <property>
+        <name>fs.viewfs.mounttable.hadoop-viewfs.link./default</name>
```

Comment:
What about that we add this to base docker images?
Then we won't need to override container's `hdfs-site.xml` with something "arbitrary" here
And we would run the test with other tests.

 @kokosing ?

Reply:
@kokosing FYI https://github.com/prestosql/docker-images/pull/38